pkgs/os-specific/linux/kernel/patches.nix

```diff
@@ -88,4 +88,18 @@
       hash = "sha256-q3iNBo8t4b1Rn5k5lau2myqOAqdA/9V9A+ok2jGkLdY=";
     };
   };
+
+  rust_1_77-6_8 = {
+    name = "rust-1.77.patch";
+    patch = fetchurl {
+      name = "rust-1.77.patch";
+      url = "https://lore.kernel.org/rust-for-linux/20240217002717.57507-1-ojeda@kernel.org/raw";
+      hash = "sha256-0KW9nHpJeMSDssCPXWZbrN8kxq5bA434t+XuPfwslUc=";
+    };
+  };
+
+  rust_1_77-6_9 = {
```

Comment:
I assume the patch is vendored because there's no stable location yet?

Reply:
Indeed.  Maybe now there is though, I'll check.

Reply:
Yeah, it's been applied to rust-next, but [it looks like that gets force pushed sometimes](https://github.com/Rust-for-Linux/linux/activity?ref=rust-next), so there's probably no stable location until it's sent to Linus.